<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.480769, y1:1, x2:0.476, y2:0, stop:0 rgba(17, 17, 17, 255), stop:1 rgba(19, 19, 19, 255));
color: rgb(212, 212, 212);</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="80">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,95">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="65,35">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.461538 rgba(255, 255, 255, 0));

border: 1px solid red;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Output log:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.461538 rgba(255, 255, 255, 0));

border: 1px solid red;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="text">
           <string>Connected: 0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="65,35">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.461538 rgba(255, 255, 255, 0));

border: 1px solid red;</string>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="45">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.461538 rgba(255, 255, 255, 0));

border: 1px solid red;</string>
            </property>
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar
{
	color: rgb(255, 255, 255);
}
QMenuBar::item::selected
{
	background-color: rgb(50, 50, 50);
}</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="styleSheet">
     <string notr="true">QMenu::item
{
	background-color: qlineargradient(spread:pad, x1:0.480769, y1:1, x2:0.476, y2:0, stop:0 rgba(17, 17, 17, 255), stop:1 rgba(19, 19, 19, 255));
    color: rgb(200, 200, 200);
}
QMenu::item:selected
{
	background-color: rgb(50, 50, 50);
	color: white;
}</string>
    </property>
    <property name="title">
     <string>Chat</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start Server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
